File: beginner/pagina12/3140/3140.cpp
#include <iostream>
#include <string>

using namespace std;

int main(){

    bool tag =  false;
    string lines;

    while(getline(cinn, line)){
        if(line.find("<body>") != string::npos){
            tag = true
            continue
        }

        if(line.find("</body>") != string::npos){
            break
        }

        if(tag){
            cout << line << endl;
        }

    }
    
    return 0;
}

File: beginner/pagina12/3140/3140.php
<?php 

    $tag = false;

    while($line = fgets(STDIN)){

        if(strpos($line, "<body>") !== false){
            $tag= true;
            continue;
        }

        if(strpos($line, "</body>") !== false){
            break;
        }

        if($tag){
            echo "$line\n";
        }
    }

?>